config/feedback-company.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'fb_client_id' => env('FB_CLIENT_ID'),
```

Comment:
Feedback company uses these terms?
Keep in mind facebook extensions might use these exact env variable names

Reply:
Feedback company uses these exact terms (client id & client secret).

I'll make the prefix a bit longer :)